content/post/practical-vbd.Rmd

```diff
@@ -108,7 +108,7 @@ TIME     <-        # Number of years to run the simulation for
 
 $$\ \frac{dSh}{dt}  = \mu_h N_h - \frac {\beta_h b}{N_h} S_h  I_v - \mu_h  S_h $$
 $$\ \frac{dIh}{dt}  = \frac {\beta_h b}{N_h}S_h I_v - (\gamma_h + \mu_h) I_h $$
-$$\ \frac{dRh}{dt}  = \gamma_h + I_h  - \mu_v I_v$$
+$$\ \frac{dRh}{dt}  = \gamma_h I_h  - \mu_h I_h$$
```

Comment:
@kcharniga, my math is not so good, but please confirm that you mean to change the addition to a multiplication here.